verde/utils.py

```diff
@@ -201,3 +203,49 @@ def maxabs(*args):
     arrays = [np.atleast_1d(i) for i in args]
     absolute = [np.abs([i.min(), i.max()]).max() for i in arrays]
     return np.max(absolute)
+
+
+def grid_to_table(grid):
+    """
+    Converts an xarray Dataset grid to a Pandas DataFrame
+
+    Parameters
+    ----------
+    grid
+        An xarray grid Dataset composed of X and Y coordinates and a variable.
+
+    Returns
+    -------
+    grid_to_table : DataFrame
+        Pandas DataFrame with coordinates and variables and increasing index
+
+    Examples
+    --------
+
+
+    """
+    coordinate_names = [
+        *grid.coords.keys()
+    ]  # extracts the coordinate names from the dataarray
```

Comment:
Kudos the wealth of comments in the code! Some of these can probably be removed because reading the code will say exactly the same as the comment. That is one of the nice things about Python. The comments can be reserved for less obvious bits of code, like the `itertools.product` below.